lib/rules/require-padding-newlines-after-blocks.js

```diff
@@ -105,6 +105,15 @@ module.exports.prototype = {
                     return;
                 }
 
+                // Check if this block is used in call expression
+                if (parentNode.parentNode.type === 'CallExpression') {
```

Comment:
Alrighty, i don't see an example when we would need exception for `parentNode.parentNode` top-level element. 

If find one, let me know